.github/workflows/build-and-publish-docs.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         env:
           SSH_DEPLOY_KEY: ${{ secrets.SSH_DEPLOY_KEY }}
         with:
-          source-directory: docs
+          source-directory: dist/pkg/github.com/pinecone-io/go-pinecone/pinecone
```

Comment:
`gopages` automatically creates a `dist` dir (and the nested dirs in this path) when run.

Reply:
the last `pinecone` dir contains a generated `index.html` file, which is what we'll push to `sdk-docs`.

Reply:
Interesting, so all the static site content is within that path after running the utility?

Reply:
It looks that way on my local, yes!